app/routes/comment.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express')
+const router = express.Router()
+const commentController = require('../controllers/comment')
+
+router.post(
+  '/',
+  commentController.comment
+)
+
+router.delete(
+  '/:id',
+  commentController.delete
+)
+
+router.patch(
+  '/:id',
+  commentController.update
+)
+
+router.get(
+  '/:id',
+  commentController.getCommentByPost
+)
+
+router.put(
+  '/upvote/:id/:userId',
+  commentController.upvote
+)
+
```

Comment:
@rak-shit Instead of making two api for upvote and downvote
you should make a single API for both